test/unit/specs/utils/attribution-html.spec.ts

```diff
@@ -0,0 +1,70 @@
+import Vuei18n from 'vue-i18n'
```

Comment:
Thank you for adding tests! The changes to the attribution HTML generation are really nice!